### DISCUSSION TITLE
How to mount Sonarr, Radarr and Lidarr using swizzin build script

### DISCUSSION THREAD

**Discussion by bleugray** [Novik/ruTorrent#2275] [Q&A]
I am receiving this message in Sonarr, Radarr and Lidarr appears to have started on Jan 17, 2022
`Unable to communicate with rTorrent (https://xxx.xxx.com/rutorrent/plugins/httprpc/action.php). Response from server does not contain valid XML`

When you navigate to https://xxx.xxx.com/rutorrent/plugins/httprpc/action.php you are presented with a blank page that says "false"

What other info would be helpful? rtorrent/rutorrent are hosted on a seedbox while Sonarr, Radarr and Lidarr are hosted locally.

v3.10

rtorrent/ruttorent are running on a seedbox that I built using swizzin

I wonder why all the apps suddenly stopped working

**Comment by stickz** [comment]
Having a value of false when you navigate to this in your web browser is expected. It doesn't mean that httprpc is not working properly. You can try disabling httprpc and simply using the rpc plugin instead. Perhaps that will work for you if these third party features are expecting a valid XML response. httprpc is actually a JSON response, which is faster but it works differently.

**Reply by stickz** [reply]
I converted this to a discussion @bleugray to preserve your question for other users. It's a great question that you asked. I don't want it to get lost when the issue is closed on GitHub. Please feel free to follow up with us, if you have any questions.

**Comment by liaralabs** [comment]
In swizzin you can also use `/<username>` as and RPC endpoint -- this is an scgi endpoint that only your user can use and is passed directly to the rTorrent socket.

https://swizzin.ltd/applications/rtorrent#rpc-mounts

**Reply by stickz** [reply]
This seems like a better solution thanks @liaralabs. This solution will allow the user to still use the httprpc plugin.

**Comment by bleugray** [comment]
Unfortunately that is exactly how it is configured now with the same result.  I have built  a new server to test and am still receiving the same error.  something changed somewhere but am unable to determine what as of yet.  Thanks for the help.

You will notice swizzin states both options are acceptable.  I have been using the _**username**_ option but began testing with the full path after the issue started.

_RPC Mount: /< username > OR /rutorrent/plugins/httprpc/action.php_

**Reply by stickz** [reply]
If you're using the **_username_** option then it's probably not related to ruTorrent.

You can try to bypass your RPC mount point with this path. `/rutorrent/plugins/rpc/rpc.php` httprpc probably won't work here.
